cave/analyzer.py

```diff
@@ -420,7 +438,7 @@ def fanova(self, incumbent, marginal_threshold=0.05):
     def local_epm_plots(self):
         plots = OrderedDict([])
         self.parameter_importance("lpi", self.incumbent, self.output)
-        for p, i in [(k, v * 100) for k, v in sorted(self.pimp.evaluator.evaluated_parameter_importance.items(),
```

Comment:
@AndreBiedenkapp why did you multiply with 100 here? It's only used in the for-loop as the `i`, which is not used.

Reply:
That was an error on my part, sorry.